src/features/Instructor/data/api.js

```diff
@@ -18,6 +18,15 @@ function getInstructorByEmail(email, options = {}) {
   );
 }
 
+function getInstructorEvents(eventData) {
```

Comment:
This is not the get method, please change the name to avoid conflicts with the service to get the  events